bundles/org.openhab.core.io.rest.sse/src/main/java/org/openhab/core/io/rest/sse/SseResource.java

```diff
@@ -262,7 +265,7 @@ public void handleEventBroadcastItemState(final ItemStateChangedEvent stateChang
         }
     }
 
-    private static class ServerAliveEvent {
-        public final String type = "ALIVE";
+    private static class KeepaliveEvent {
+        public final int interval = KEEPALIVE_INTERVAL_SECONDS;
```

Comment:
Please keep the type field as it is expected in any message. 
And if you keep value "ALIVE", it would avoid changing again what was done after @J-N-K change.

Reply:
Who is expecting this message?  If you read the thread above it sounds like the assumption is that the UI's are not aware of this message?  Maybe you could explain how UIs use the "ALIVE" message ?

Reply:
Until now, all messages sent by openHAB have a type field.
The type was used to distinguish an ALIVE message from any other "standard" message.
This is used in the remote openHAB binding.
Basic UI is using the same kind of check but this is for sitemap SSE messages (so not applicable here). But htose SSE messages also have a type field.
I don't know what is the standard with SSE message but having a type looks like a good idea to me.

Reply:
I would also suggest to keep the `type` and add the `interval` as additional field. So everyone not interested (already filtering out `ALIVE` currently) does not have to change the code. And I agree that always having a type for an event sounds like a good idea.

Reply:
I will refactor this again and merge the two concepts.

> And I agree that always having a type for an event sounds like a good idea.

I agree, but SSE connections have an event name attribute which allows you to specify the type of event, right now the `ALIVE` message is sending an event type of `event` (event->event ?)  which is a little confusing, and why i called my event `keepalive` since that is the event type of the object i was sending.  

Since it sounds like there are legacy clients who depend on the event->event->ALIVE message, i will use this as well and add the additional interval field as well as turning up the frequency.  I'll update the PR shortly.